Docker/base-images/entando-eap71-base/README.md

```diff
@@ -0,0 +1,9 @@
+The sole purpose of this image is to allow new developers to get an Entando 
+environmet up and running as easily as possible, and to get quick feeback from
+tweaking the Maven project.
```

Comment:
Should this README include simple instructions for running this image?

Reply:
Nope. It is more like an 'abstract' image. Only to be used from an extending image. It is extended either through the maven docker plugin, or by the S2I process. That being said, I still need to touch up all of the new README's to make all of this clearer